styleguide.md

```diff
@@ -0,0 +1,13 @@
+# Styleguide
```

Comment:
@thelostone-mc Mind adding this file to the `pydocmd.yml` file? Probably under the `General` section somewhere.